### DISCUSSION TITLE
Printed upgrades

### DISCUSSION THREAD

**Discussion by bassamanator** [bassamanator/everything-sovol-sv06#30] [General]
Feel free to share printable upgrades here.

Find my recommendations [here](https://github.com/bassamanator/everything-sovol-sv06/blob/main/parts/README.md#printed-parts).

**Comment by albertoabrao** [comment]
Not a part, but **I strongly suggest that you open your mainboard enclosure and wire the fan for the board together with the fan for the throat (that's the fan to the left of the extruder, and it's always on). The SV06 board has its fan together with the part cooling fan. If that's running at 0%... board is not being cooled!**

Also, here's some prints that were so good, I had to re-print them for my second SV06. **All in PETG!**

## **These are a must!** Trust me, you *do not want to*  re-solder the thermistor wire on the heated bed...

[Heatbed Cable Support for SOVOL SV06 / SV06+ 3d printer](https://www.printables.com/model/409689-heatbed-cable-support-for-sovol-sv06-sv06-3d-print)
[Raised Cable Strain Relief for SOVOL SV06 3d printer](https://www.printables.com/model/461239-raised-cable-strain-relief-for-sovol-sv06-3d-print)

### Now, print this, and measure according to the instructions. If it doesn't work, change the flow on your slicer.
[Flow Calibration Cube](https://www.printables.com/model/81314-flow-calibration-cube)

### And then, print this. Make sure printer is calibrated before, so that you get a satisfying "click" when clipping this on to the X axis assembly.
[Back mounted cooling fan duct for SOVOL SV06 3d printer with 4010 blower fan](https://www.printables.com/model/447126-back-mounted-cooling-fan-duct-for-sovol-sv06-3d-pr)

### I also consider this one essential:
[Belt tension tester](https://www.printables.com/model/447036-belt-tension-tester)

## Now for everything else...
[SV06 Plus Z Axis leveling tool](https://www.printables.com/model/529401-sv06-plus-z-axis-leveling-tool) - although I use a non-Plus SV06, these are taller (you can scale them as needed too, of course). I don't use the bed to level the Z Axis, opting to have them on top of the mounts for the Z screw stepper motors on each side.

[Sovol SV06 simple Zaxis Bearing Adapter](https://www.printables.com/model/462039-sovol-sv06-simple-zaxis-bearing-adapter) - Use this with one of those [flexible couplers](https://www.amazon.ca/AFUNTA-Flexible-Couplings-Compatible-Machines/dp/B07JL1QYLS/ref=sr_1_6?crid=1W3GQUX9QEE7X&keywords=flexible+coupler&qid=1693798663&s=industrial&sprefix=flexible+couple%2Cindustrial%2C137&sr=1-6).

[Sovol SV06 608 Bearing Filament Runout Sensor Mount](https://www.printables.com/model/423624-sovol-sv06-608-bearing-filament-runout-sensor-moun) - you can screw the spool holder back on top of it.

[SV06 Clip-On Bed Aligner](https://www.printables.com/model/380404-sv06-clip-on-bed-aligner)
